freqtrade/exchange/kraken.py

```diff
@@ -156,3 +156,25 @@ def _get_params(self, ordertype: str, leverage: float, time_in_force: str = 'gtc
         if leverage > 1.0:
             params['leverage'] = leverage
         return params
+
+    def _get_funding_fee(
+        self,
+        size: float,
+        funding_rate: float,
+        mark_price: float,
+        time_in_ratio: Optional[float] = None
+    ) -> float:
+        """
+            Calculates a single funding fee
+            :param size: contract size * number of contracts
+            :param mark_price: The price of the asset that the contract is based off of
+            :param funding_rate: the interest rate and the premium
+                - interest rate:
+                - premium: varies by price difference between the perpetual contract and mark price
+            :param time_in_ratio: time elapsed within funding period without position alteration
+        """
+        if not time_in_ratio:
+            raise OperationalException(
+                f"time_in_ratio is required for {self.name}._get_funding_fee")
+        nominal_value = mark_price * size
+        return nominal_value * funding_rate * time_in_ratio
```

Comment:
Krakens formula is a bit different, I included it in here to save whoever some time when they decide they want to trade futures on kraken, but there's a few tests that I had to comment out, due to `calculate_funding_fees` not and `update_funding_fees` not passing in the `time_in_ratio` parameter that kraken needs. I wrote tests for this method, but probably won't implement the changes needed in `calculate_funding_fees` not and `update_funding_fees` to trade futures on Kraken

Reply:
please add a comment in the docstring mentioning that it's intentionally not active ... 

I usually don't check comments on PR's when i wonder about a line of code ... or if i do, only hours into the investigation, and i assume others will do the same :laughing: